packages/style-utilities/src/utilities/icons.ts

```diff
@@ -8,6 +8,11 @@ export interface IIconSubset {
   };
 
   style?: IRawStyle;
+  /**
+   * Whether or not to merge any props required for images into the child elements for
+   * accessibiity comlpliance.
```

Comment:
Just a personal preference, but I found this docs wording a little confusing -- would it be possible to update it to something like "Force the icon to act like an image icon even when not using `imageProps` or `IconType.image`."?